### PR TITLE
ZCS-14375: update translation for 2FA

### PR DESCRIPTION
**Pull request by k-kato**
Adding translation for keys related to two-factor authentication.

Related PRs:
* https://github.com/Zimbra/zm-web-client/pull/848
* https://github.com/Zimbra/zm-web-client/pull/850

### CODE REVIEW

**Review by sitaram-mulik-synacor — APPROVED** [review]

File: WebRoot/messages/ZmMsg.properties [hunk 1]
```diff
@@ -4626,7 +4626,7 @@ recoveryEmailErrorMessageInvalidCode = Verification of recovery email address ve
 passwordRecoveryTitle= Set up Password Recovery
 passwordRecoveryButtonCancel= Back to sign in
 passwordRecoveryGetAccountDescription = Please enter your username so we may retrieve the recovery email you have on record. \
-<br><br>Don't have a recovery email on record? Please contact your system administrator for help resetting your password.
+<br><br>Don\u2019t have a recovery email on record? Please contact your system administrator for help resetting your password.
```
Comment (silentsakky) [inline WebRoot/messages/ZmMsg.properties:4629]: not sure if we need any encoding in english messages

Comment (k-kato) [reply WebRoot/messages/ZmMsg.properties:4629]: I don't know the exact reason, but a single quote `'` has been replaced with `\u2019` on the following PR.
* https://github.com/Zimbra/zm-web-client/pull/134

Comment (silentsakky) [reply WebRoot/messages/ZmMsg.properties:4629]: ok got it, I think without the encoding files were not loading properly

**Review by silentsakky — APPROVED** [review]

**Comment by k-kato** [comment]
rebased

**Pull request merged**